default.json

```diff
@@ -19,6 +19,7 @@
   "rebaseWhen": "behind-base-branch",
   "platformAutomerge": false,
   "internalChecksFilter": "strict",
+  "labels": ["Dependencies"],
```

Comment:
I'm going to give this a quick test in Scan Files since the label might need to already exist in the repo to work.

Reply:
According to Renovate it autocreates it but definitely need to see a new PR in action to fully test. Good idea.

Reply:
Tested and it does indeed auto add the label!